app/design-system/design-system-hedvig/src/main/kotlin/com/hedvig/android/design/system/hedvig/tokens/StepperTokens.kt

```diff
@@ -0,0 +1,88 @@
+package com.hedvig.android.design.system.hedvig.tokens
+
+import androidx.compose.ui.unit.dp
+
+internal object StepperColorTokens {
+  val ContainerColor = ColorSchemeKeyTokens.SurfacePrimary
+  val EnabledSymbolColor = ColorSchemeKeyTokens.FillPrimary
+  val DisabledSymbolColor = ColorSchemeKeyTokens.FillDisabled
+  val TextColor = ColorSchemeKeyTokens.TextPrimary
+  val LabelColor = ColorSchemeKeyTokens.TextSecondaryTranslucent // todo: should we use Translucent?
```

Comment:
Yeah you are right, I doubt it's what we actually want here. I would start off with the normal opaque color and if it turns out to be a problem we'll reconsider